doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -4618,6 +4618,25 @@ The fully expanded example above (without environment variables) looks like this
 
   curl "https://demo.dataverse.org/api/info/settings/:MaxEmbargoDurationInMonths"
 
+Get Export Formats
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Get the available export formats. The response contains a list of objects each containing the display name and format name for an available exporter.
```

Comment:
Does this include [external exporters](https://guides.dataverse.org/en/6.3/installation/advanced.html#installing-external-metadata-exporters)? I assume so. Can we please mention they are included as well?

Reply:
As far as I know, all exporters, standard and custom, are managed by the `ExportService`, and are consequently all listed. I improved the docs in [3b2c8e5](https://github.com/IQSS/dataverse/pull/10739/commits/3b2c8e5aeda5e401e60586b239bae7e5285f36fd).